### DISCUSSION TITLE
Add new HtmlBody Editor

### DISCUSSION THREAD

**Discussion by jsalgado10** [OrchardCMS/OrchardCore#7948] [Q&A] [answered]
How would I go about adding a new Editor to the htmlbody Part? Currently trumbowyg and wysiwyg are the only options. I would like to add other editor.

I have created 2 views on my Theme, but they are still not showing on the drop down list
HtmlField-MyEditor.Edit.cshtml
HtmlField-MyEditor.Option.cshtml

I have also tried creating 2 views with htmlbodypart instead of htmlfield on the filename, but that still is not working.

Do I need to create a new module for this?

**Comment by deanmarcussen** [comment, marked as answer]
You will need to place these in a module.

When they are in a theme they are not active, because you are trying to use them in the admin, which is themed by `TheAdmin` theme, not your theme, so the views are not being found. 

Example here for CKEditor https://github.com/ThisNetWorks/ThisNetWorks.OrchardCore.CKEditor

Also worth taking a dependency on `OrchardCore.Html` in the manifest for the module https://github.com/ThisNetWorks/ThisNetWorks.OrchardCore.CKEditor/blob/60bb02fac13ff3b2c7ecdf75537452641b0b1c87/src/ThisNetWorks.OrchardCore.CKEditor/Manifest.cs#L9

**Reply by jsalgado10** [reply]
This is exactly what I needed. I will create my own module and add my new html editor
